.github/workflows/preview-publish.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish preview
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
+      - unlabeled
+    paths-ignore:
+      - '**/*.md'
+
+concurrency:
+  group: preview-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+env:
+  CC_CLEVER_TOOLS_PREVIEWS_CELLAR_BUCKET: ${{ vars.CC_CLEVER_TOOLS_PREVIEWS_CELLAR_BUCKET }}
+  CC_CLEVER_TOOLS_PREVIEWS_CELLAR_KEY_ID: ${{ secrets.CC_CLEVER_TOOLS_PREVIEWS_CELLAR_KEY_ID }}
+  CC_CLEVER_TOOLS_PREVIEWS_CELLAR_SECRET_KEY: ${{ secrets.CC_CLEVER_TOOLS_PREVIEWS_CELLAR_SECRET_KEY }}
+
+jobs:
+  build:
+    if: ${{ !startsWith(github.event.pull_request.head.ref, 'release-please--') }}
+    uses: ./.github/workflows/build.yml
+    with:
+      version: ${{ github.event.pull_request.head.ref }}
+      include-windows: ${{ contains(github.event.pull_request.labels.*.name, 'build:win') }}
```

Comment:
question: why do you introduce a condition for the windows build?

I think we should always perform the build for every platform. I consider this build process as part of our test process. We can't wait for the publish process to validate that the build process is still valid.